tests/unit/components/__snapshots__/HdIcon.spec.js.snap

```diff
@@ -2,16 +2,16 @@
 
 exports[`HdIcon adds title and desc tag to the svg 1`] = `
 <svg aria-describedby="icon-id-desc" height="50" width="50" viewBox="222 126 53 53" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" id="icon-id">
-  <path class="color1" d="M272.48 137.62L258.61 150.5L244.75 141.58L254.65 126.73L272.48 137.62Z" fill="#1895FF"></path>
-  <path class="color2" d="M242.22 176.24L222.97 160.68L230.47 145.1L243.56 155.68L242.22 176.24Z" fill="#1C3553"></path>
-  <desc id="icon-id-desc">Some description.</desc>
-  <title>This make me accessible.</title>
+  <path xmlns="http://www.w3.org/2000/svg" class="color1" d="M272.48 137.62L258.61 150.5L244.75 141.58L254.65 126.73L272.48 137.62Z" fill="#1895FF"></path>
```

Comment:
Is there any spesific reason to add `xmlns="http://www.w3.org/2000/svg"` to the `path` element?